### PR TITLE
Update text in 'Add volume', 'Edit volume metadata' modals

### PR DESCRIPTION
**Pull request by hstastna**
## 📝 Description

Update various texts in _Add volume_, _Edit volume metadata_ modals:

In _Add volume_ modal, accessible from both _Bootable volumes_ list and _Catalog > InstanceTypes_, change the following:
- _Clone existing PVC_ checkbox popover text to: "This will create a cloned copy of the PVC in the destination project."
- add `HelpIcon` with popover text to _Volume metadata_: "Set the volume metadata to use the volume as a bootable image."
- update _Preference_ popover text by adding "Read more" external link to:
  https://kubevirt.io/user-guide/virtual_machines/instancetypes/#virtualmachinepreference

In _Edit volume metadata_ accessible from _Bootable volumes list_, change the following:
- update _Preference_ popover text to "The preferred VirtualMachine attribute values required to run a given workload."
  and add "Read more" external link (same as in _Add volume_ modal)
- change _Default InstanceType_ popover text to: "The default InstanceType for this volume."

Also update the text of _Delete_ button (accessible from _Bootable volumes_ list) to:
"Only the bootable metadata will be deleted"

Make these all updates according to the last design changes and consistency reasons.

_Related design docs:_
https://docs.google.com/document/d/1HTF6_H2WDXwlVy7RnrHH28dzFlhNIa15Hhpeoyg2R0Q/edit#heading=h.xt31tt3xq1x
https://docs.google.com/document/d/15Cv54978AZKeukkurvkFN1YEx_kBsm_3XcmLey_cgZY/edit#heading=h.5tnevt9lwj7z

## 🎥 Screenshots
**Before:**
_Clone existing PVC_ checkbox popover text:
clone_before
 _Preference_ popover text and no any popover text or "?" icon for _Volume metadata_:
pref_before
_Edit volume metadata_ modal:
edit1_before
edit2_before
 _Delete_ button in _Bootable volumes_ list:
btn_before

**After:**
_Clone existing PVC_ checkbox popover text:
clone_after
Updated  _Preference_ popover text:
pref_after
Added popover text with "?" icon for _Volume metadata_:
vol_after
_Edit volume metadata_ modal:
edit1_after
edit2_after
 _Delete_ button in _Bootable volumes_ list:
btn_after

### CODE REVIEW

**Comment by hstastna** [comment]
@avivtur @metalice @pcbailey @upalatucci @vojtechszocs please review

File: src/views/bootablevolumes/actions/hooks/useBootableVolumesActions.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const useBootableVolumesActions: BootableVolumesActionsProps = (
       id: 'delete-bootablevolume',
       disabled: !canUpdateDataSource,
       label: t('Delete'),
-      description: 'Only the labels data will be deleted',
+      description: 'Only the bootable metadata will be deleted',
```
Comment (metalice) [inline src/views/bootablevolumes/actions/hooks/useBootableVolumesActions.tsx:53]: should this be also translated?

**Pull request merged**